dev/com.ibm.ws.logging/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-    Copyright (c) 2011, 2020 IBM Corporation and others.
+    Copyright (c) 2011, 2021 IBM Corporation and others.
```

Comment:
No need to change the copyright year as there is no change in the file.

Reply:
Resolved. It was leftover after the metatype documentation was temporarily removed.